app/views/admin/users/unconfirmed.html.erb

```diff
@@ -1,6 +1,6 @@
 <h2>Unconfirmed Members</h2>
 
-<p>There are currently <%= pluralize(@users.count, 'unconfirmed member') %></p>
+<p>There are currently <%= pluralize(User.unconfirmed.count, 'unconfirmed member') %></p>
```

Comment:
Why is @users.count not giving the right figure? Is it just giving the amount being shown on the page? That surprises me.

Reply:
Yes, it's just giving the amount being shown on the page, this is the same as the confirmed page's code